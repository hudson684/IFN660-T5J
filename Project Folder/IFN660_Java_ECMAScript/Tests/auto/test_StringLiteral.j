""
"a"
"abc"
"abcdefgh"
"\""
"\t"
"\7"
"\77"
"\177"
"\355"
"this is a string"