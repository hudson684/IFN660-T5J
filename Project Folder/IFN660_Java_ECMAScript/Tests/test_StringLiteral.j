""
"a"
"abc"
"abcdefgh"
"\r"
"\t"
"\7"
"\77"
"\177"
"\555"