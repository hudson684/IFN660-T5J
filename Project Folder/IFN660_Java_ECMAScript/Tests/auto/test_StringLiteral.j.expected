StringLiteral ("")
StringLiteral ("a")
StringLiteral ("abc")
StringLiteral ("abcdefgh")
StringLiteral ("\"")
StringLiteral ("\t")
StringLiteral ("\7")
StringLiteral ("\77")
StringLiteral ("\177")
StringLiteral ("\355")
StringLiteral ("this is a string")
EOF